packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -121,10 +123,10 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
         .pipe(first())
         .subscribe(
           (response) => {
+            getVisitFormData.currentVisit = response.data;
+            localStorage.setItem('visit-form', JSON.stringify(getVisitFormData));
```

Comment:
I've said this before, but please don't use `localStorage` for tracking temporary data. We really don't need any storage here and could likely just set a variable value, but if we _must_ use storage for this, use sessionStorage (local storage is for values that persist across sessions—not at all what we want here)